sim/battle-stream.ts

```diff
@@ -39,8 +39,8 @@ function splitFirst(str: string, delimiter: string, limit: number = 1) {
 }
 
 export class BattleStream extends Streams.ObjectReadWriteStream<string> {
-	readonly debug: boolean;
-	readonly keepAlive: boolean;
+	debug: boolean;
+	keepAlive: boolean;
```

Comment:
errr

Reply:
You had wanted this in the previous change?

Reply:
Yeah but I thought I merged it in the previous change.

Reply:
https://github.com/Zarel/Pokemon-Showdown/blob/master/sim/battle-stream.ts#L42 still there on HEAD